marklogic-data-hub-central/ui/src/pages/TilesView.test.tsx

```diff
@@ -82,25 +56,49 @@ describe('TilesView component', () => {
         expect(getByText('Customer')).toBeInTheDocument();
     });
 
+    test('Verify Load tile displays from toolbar with readIngestion authority', async () => {
+        const authorityService = new AuthoritiesService();
+        authorityService.setAuthorities(['readIngestion']);
+        const {getByLabelText, getByText, queryByText} = render(<AuthoritiesContext.Provider value={authorityService}><TilesView/></AuthoritiesContext.Provider>);
+
+        // Curate tile not shown initially
+        expect(queryByText("icon-load")).not.toBeInTheDocument();
+        expect(queryByText("title-load")).not.toBeInTheDocument();
+
+        await curateData.setupMockAPIs(axiosMock);
+        fireEvent.click(getByLabelText("tool-load"));
```

Comment:
Can we also add a test to verify readIngestion can not access Model, Curate and Run to ensure coverage per test plan.

Reply:
added the check for access to other tiles.